xray/src/generation.rs

```diff
@@ -272,6 +272,50 @@ impl HeightStddevColoringStrategy {
     }
 }
 
+/// Build a parent image created of the 4 children tiles. All tiles are optionally, in which case
```

Comment:
Nit: why three slashes? Is it a Rust thing? I've never seen that before.

Reply:
https://doc.rust-lang.org/beta/rustdoc/what-is-rustdoc.html

Rust has something like doxygen build in.